common/lib/conftest.py

```diff
@@ -21,3 +26,16 @@ def no_webpack_loader(monkeypatch):
         "webpack_loader.utils.get_files",
         lambda entry, extension=None, config='DEFAULT', attrs='': []
     )
+
+
+def pytest_configure(config):
```

Comment:
Is there a common code location that `pytest_configure` could live in order to DRY this PR out a bit?

Reply:
There wasn't an obvious choice at first, but I think I can find a home for that.